packages/clerk-js/src/ui/components/CreateOrganization/CreateOrganizationForm.tsx

```diff
@@ -87,6 +89,11 @@ export const CreateOrganizationForm = withCardStateProvider((props: CreateOrgani
 
       void userMemberships.revalidate?.();
 
+      if (sessionTasksContext) {
+        await sessionTasksContext.nextTask();
```

Comment:
This is a pretty similar order as we'd have for custom flows: 

1. Create an organization 
2. Update it to the active session 
3. Verify if there are remaining tasks, and if so, navigate to the next one. 

I've moved it inside of here now instead of passing it within `onComplete` prop so that we can `await` for it. `onComplete` is a sync property and I don't think we should turn into a promise in order to avoid side effects.